.github/workflows/bench_pr.yml

```diff
@@ -13,9 +13,9 @@ concurrency:
   cancel-in-progress: true
 
 # this is needed to gain access via OIDC to the S3 bucket for caching
-permissions:
-  id-token: write
-  contents: read
+permissions: write-all
+#  id-token: write
+#  contents: read
```

Comment:
@kevinnassery is this safe to do?

Reply:
I'm ok with it if it's a temporary debugging tool based on Morgan's comment, however - if it solves the issue, then we should proceed identifying which of the granular job permissions are required.

Reply:
For temporary debugging I'm ok with it, in the long term the granular permission table should be used:
https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

It might be faster to simply enumerate the specific permissions that could be killing the job (id-token, actions, contents) seem most likely.

Reply:
Temporary debugging was the intention, it would never be given these permissions.